app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -20,6 +20,31 @@ class Api::V2::ContentViewsController < Api::V2::ApiController
       param :component_ids, Array, :desc => N_("List of component content view version ids for composite views")
     end
 
+    api :POST, "/content_views/:id/export", N_("Export a content view to disk")
+    param :id, :identifier, :desc => N_("Content view identifier"), :required => true
```

Comment:
I was a little up in the air about this. Let's say I do my initial "base" export off of CVV version 5. I then want the incremental, would I need to get the export time of version 5, then do an incremental based on that of say, version 8?

I agree that CVV makes more sense here, but I feel like CVVs and date-based incrementals are mixing metaphors in some sense. I'm not wedded to this though so if you think its OK then I will change to CVV :bride_with_veil: